tests/lua-tests/src/helper.lua

```diff
@@ -1,4 +1,4 @@
-require "Cocos2d"
+require "cocos/Cocos2d"
```

Comment:
Why change path?

Reply:
It seems many codes changes the path, i don't list all the modifcations.

Reply:
modified